nautobot_golden_config/tests/test_views.py

```diff
@@ -0,0 +1,47 @@
+"""Unit tests for nautobot_golden_config views."""
+
+from unittest import skip
+from django.test import TestCase
+
+from nautobot.dcim.models import Device
+from nautobot_golden_config import views, models
+
+from .conftest import create_feature_rule_json, create_device_data
+
+
+class ConfigComplianceOverviewOverviewHelperTestCase(TestCase):
+    """Test ConfigComplianceOverviewOverviewHelper."""
+
+    def setUp(self):
+        """Set up base objects."""
+        create_device_data()
+        self.dev01 = Device.objects.get(name="Device 1")
```

Comment:
Is this a typical pattern?  Confused why only dev01 is in the `self` instance.

Reply:
@scetron what about this one 😄

Reply:
😅 , I am not sure if this was left over from my inspiration, or what.  I believe since it's in the setup, it would make it available for further unit tests, but I could be wrong. Additionally, it should be all of the devices or none of them, so I will remove the self for now.